src/components/PostPageDetail/index.js

```diff
@@ -67,9 +65,9 @@ const PostPageDetailIdComponent = (props) => {
   const [statusUpvote, setStatusUpvote] = React.useState(false);
   const [statusDownvote, setStatusDowvote] = React.useState(false);
   const [loadingPost, setLoadingPost] = React.useState(false);
-  const [item, setItem] = React.useState(null);
   const navigation = useNavigation();
   const route = useRoute();
+  const [item, setItem] = React.useState(route?.params?.data);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The initialization of `item` state has been changed to use `route.params.data`. Make sure that `route.params.data` always contains the expected data structure for `item`.
```diff
- const [item, setItem] = React.useState(null);
+ const [item, setItem] = React.useState(route?.params?.data);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->